### ISSUE TITLE
Primary bot should be managed by bot registry

### ISSUE THREAD

**Issue by psrpinto** [Automattic/neurobot#91]
Primary bot is `@neurobot`

- [x] Use same mechanism as other bots to configure primary bot (seeds + .env)
- [x] Insert in `bots` table
- [x] Add to bot registry

**Comment by ashfame** [comment]
> Insert in bots table with primary = true

Do we add a primary column on bots table now? OR just treat the primary bot, that's imported automatically from .env to have ID:1 always? I am leaning towards latter.

**Comment by psrpinto** [comment]
My preference would be to have a `primary` column, for multiple reasons:

1. it's more flexible, e.g. allows for the primary bot to be changed
2. it's more explicit since doesn't rely on ID 1 having special meaning, someone looking at the table immediately know which bot is primary
3. We'd automatically have a `Primary` field on the `Bot` object. Otherwise we'd need to add a `IsPrimary()` method, which I guess would also be fine.

On the other hand, relying on ID 1 has the advantage of us not needing to enforce that no more than one bot has `primary = true`.


In summary, both options work for me.

(BTW, I added an extra step to the issue description (Use same mechanism as other bots to configure primary bot))

**Comment by ashfame** [comment]
Sure! Let's go the flexible route. Have at it :)

**Comment by psrpinto** [comment]
After further discussion, @ashfame and I decided to proceed with using ID 1 for the primary bot.
